core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/internal/DefaultSyncSignedRequest.java

```diff
@@ -21,10 +21,10 @@
 import software.amazon.awssdk.utils.ToString;
 
 @SdkInternalApi
-public final class DefaultSyncSignedHttpRequest
-    extends DefaultSignedHttpRequest<ContentStreamProvider> implements SyncSignedHttpRequest {
+public final class DefaultSyncSignedRequest
```

Comment:
you missed changing the toString() here. double check by searching through these modules with "Http" to make sure all are covered.